classes/profile.php

```diff
@@ -379,32 +380,33 @@ protected static function define_properties(): array {
         return [
             'reason' => ['type' => PARAM_INT],
             'scripttype' => ['type' => PARAM_INT],
-            'method' => ['type' => PARAM_ALPHA],
-            'created' => ['type' => PARAM_INT],
+            'method' => ['type' => PARAM_ALPHA, 'default' => ''],
```

Comment:
will ci complain if we index these all the line up in columns?

Reply:
It does.